test/parallel/test-https-agent-unref-socket.js

```diff
@@ -6,8 +6,24 @@ if (!common.hasCrypto)
 
 const https = require('https');
 
-const request = https.get('https://example.com');
+if (process.argv[2] === 'localhost') {
+  const request = https.get('https://localhost:' + process.argv[3]);
 
-request.on('socket', (socket) => {
-  socket.unref();
-});
+  request.on('socket', (socket) => {
+    socket.unref();
+  });
+} else {
+  const net = require('net');
+  const server = net.createServer();
+  server.listen(0);
+  server.on('listening', () => {
+    const port = server.address().port;
+    const { fork } = require('child_process');
+    const child = fork(__filename, ['localhost', port], {});
+    child.on('close', (exit_code) => {
+      server.close();
+      if (exit_code == null || exit_code !== 0)
+        process.exit(1);
```

Comment:
Any reason for not using `assert.strictEqual(exit_code, 0);` ?

Reply:
Thank you for the hit. I'll change this :+1: